db/seeds.rb

```diff
@@ -1,7 +1,16 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+require 'faker'
+
+User.create(email: "admin@email.com",
+            encrypted_password: "$2a$10$8IJ8lAADmoL0yU6no0BaJOH2P2hSkmcS4",
+            sign_in_count: 1,
+						created_at: "2015-10-05 15:14:54.746525",
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Tab detected.